snuba/web/query.py

```diff
@@ -229,6 +231,23 @@ def record_missing_use_case_id(request: Request, dataset: Dataset) -> None:
             )
 
 
+def record_subscription_created_missing_tenant_ids(request: Request) -> None:
+    """
+    Used to track how often new subscriptions are created without Tenant IDs.
+    """
+    if request.referrer == SUBSCRIPTION_REFERRER:
+        if not (tenant_ids := request.attribution_info.tenant_ids):
+            metrics.increment("subscription_created_without_tenant_ids")
```

Comment:
Why would it ever be the case? Did we not update the code in Sentry to send them already?

Reply:
We have Tenant IDs on almost everything _except_ subscriptions because we didn't go down the route of updating subscriptions during capacity management work. Executor queries have no Tenant IDs yet because of this.